locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml

```diff
@@ -122,22 +122,22 @@ de:
           s: 22
           from: 00:02:20,22
           to: 00:02:25,00
-          text: The strikes of 1936 were the beginning of it all.
+          text: Mit den Streiks 1936 fing alles an.
         sequence-23:
           s: 23
           from: 00:02:25,00
           to: 00:02:33,27
-          text: There was a labour movement that manifested itself. I was already
-            part of the working class.
+          text: Daraufhin bildete sich die Arbeiterbeweung heraus. Zu dieser Zeit war ich
+          bereits Teil des Arbeiterumfelds.
```

Comment:
For me the term `Arbeiterumfelds` doesn't sound correct. The English translation `working class` is also not perfect in my opinion. How about sticking to the French and use milieux like `Arbeitermilieus`?